examples/centralized_design_autoscale/example.tfvars

```diff
@@ -576,6 +576,23 @@ vmseries_asgs = {
       }
     }
 
+    launch_template_version = "1"
```

Comment:
If there will be a change in launch template we will be still using version 1 not the latest one with the changes.

Reply:
Ok, thanks for the clarification, I was not aware that this is an issue. This is not explicit example for the instance refresh, I would propose to leave like this and give an information about it like a comment or so, to clarify it, are you ok with this approach?

Reply:
Ok, done.